test/distributions/bradc/assoc/UserMapAssoc.chpl

```diff
@@ -508,6 +508,12 @@ class UserMapAssocDom: BaseAssociativeDom {
   proc dsiReprivatize(other) {
     locDoms = other.locDoms;
   }
+
+  proc dsiDestroyDom() {
+    for localeIdx in dist.targetLocDom do
```

Comment:
Why is this a 'for' while destroyArr uses a coforall?

Reply:
Good point, not sure why I did that...

Reply:
The pull of the leak-free side is strong and hard to resist.

Reply:
Is it that way in something you based it off of, Block etc?

Reply:
Ah, I think I based it on the loop in UserMapAssocDom.setup. Anyways, I'll update it to use a coforall and re-run tests on that directory.

Reply:
I asked because I wondered if whatever you based it on should use coforall.

Reply:
Maybe worth a comment if it's not obvious.